content.md

```diff
@@ -1,125 +1,126 @@
+- **Replaced previous, high-toil on-prem system management** with automatic workload scheduling. *(Think proto-containerization with Linux VMs and specialized workloads.)* 
+- Lead on **isolating different customers' monitoring** by sharding workloads from large, shared physical servers onto small VMs, with scheduling based on configured resource constraints. 
+- Established **standard system configuration best practices** applied across 500+ servers and all team services. 
+ 
+#### Google Apps for Your Domain (Now GSuite) Administrator Lead for google.com - *(2008 - 2010)* 
+- **Managed all domain-wide policies** for Google's own GSuite domain, including: 
+  - **User status/password sync** from the various systems of record. 
+  - Populating **IP whitelist by digesting internals network configs**. 
+  - **Group/policy synchronization.** 
+- **Set policy for access to former user data**, in line with established Google privacy/authorization practices. 
+- **Maintained internal tool for providing delegated, logged administrative access** to internal helpdesk, including managing quota and access to orphaned data. 
+- **Met in person with various GSuite teams** (Calendar, Admin Console, Gmail, Docs, etc.) PMs to **provide feedback on missing features and requirements for enterprise** use cases, helping steer roadmaps. 
+- Worked with GSuite team to **roll out Google multi-login**, including both managing the google.com domain changes and consulting on requirements for the feature's roll out. Received an **OC Award Honorable Mention**. 
+ 
+#### Single Sign-On SRE "Tech Lead" (Team of 1) - *(2008 - 2010)* 
+- Provided SRE support for Google's in-house SSO system, including monitoring, alerting, and deployment support. 
+ 
+### Google Inc. - **Sysops Intern** - *June 2006 - August 2006* 
+- **Migrated Googler Calendars from Oracle Calendar to Google Calendar** 
+    - In advance of Google's initial release of Google Calendar for enterprise and to aid with internal dogfooding, I wrote all the tools to export all employee calendar data from Oracle Calendar and import it into Google Calendar. 
+- **Global logs visualization** 
+    - Used a combination of Sawzall and Python to analyze and geolocate logs data from internal infrastructure and visualize the magnitude of internal traffic onto Google Earth using dynamically rendered KML. 
+ 
+Volunteering/Service 
+-------------------- 
+    
+### CSSI@Google LAX 
+ 
+[CSSI](https://buildyourfuture.withgoogle.com/programs/computer-science-summer-institute/) is a three-week intensive programming course for rising college freshman, primarily from groups underrepresented in tech. It culminates in students designing, implementing, and building their own web app. 
+ 
+#### Co-Lead Instructor - Summer 2018 and 2019 
+- **Ran Google L.A.'s CSSI instruction** for two summers, along with my co-lead. 
```

Comment:
I think you can strike “along with co-lead”

Reply:
My worry is, world being small and tech being incestuous, I don't want Michael to think I'm taking credit for his work, and also don't want anybody who knows Michaels CSSI work to be like, "this guy's lying".  Removed the "along with..." and changed the verb to Co-Ran.